File: Readio/src/main/java/ninegle/Readio/category/mapper/CategoryMapper.java
package ninegle.Readio.category.mapper;

import java.util.ArrayList;

import ninegle.Readio.category.dto.CategoryGroupDto;

public class CategoryMapper {

	public static CategoryGroupDto toCategoryGroupDto(long id, String major) {
		return CategoryGroupDto.builder()
			.id(id)
			.major(major)
			.subs(new ArrayList<>())
			.build();
	}
}

File: Readio/src/main/java/ninegle/Readio/category/mapper/CategoryMapper.kt
package ninegle.Readio.category.mapper

import ninegle.Readio.category.dto.CategoryGroupDto

fun toCategoryGroupDto(id: Long, major: String): CategoryGroupDto {
    return CategoryGroupDto(
        id = id,
        major = major,
        subs = mutableListOf()
    )
}
